### DISCUSSION TITLE
how do i install java on React OS

### DISCUSSION THREAD

**Discussion by broman2323** [copy/v86#841] [General]
can somebody help me or just give me a save state that has that installed on that OS

**Comment by spetterman66** [comment]
WHY? it's better to run it on Arch Linux

**Comment by SuperMaxusa** [comment]
I managed to run OpenJDK 1.8.0 (**8u262-b10**) in ReactOS (I haven't tested other versions yet).

![1](https://user-images.githubusercontent.com/41739128/234255837-f715826c-6e7d-4f6c-b23b-6a6e72d2e473.png)

I used the version from [OpenLogic](https://www.openlogic.com/openjdk-downloads?field_java_parent_version_target_id=416&field_operating_system_target_id=436&field_architecture_target_id=386&field_java_package_target_id=396) (select **.zip**), then unpacked the archive and added the path to the `bin` folder in the `PATH` enviroment variable (`set PATH=%PATH%;C:\path_to_jdk\bin`)

**Comment by Darin755** [comment]
This might be outside the scope of this project. I would ask on the React os forms
